pydoctor/epydoc/markup/restructuredtext.py

```diff
@@ -250,9 +273,18 @@ def _add_field(self,
             lineno: int
             ) -> None:
         field_doc = self.document.copy()
-        for child in fbody: field_doc.append(child)
-        field_pdoc = ParsedRstDocstring(field_doc, ())
-        self.fields.append(Field(tagname, arg, field_pdoc, lineno - 1))
+        for child in fbody: 
+            field_doc.append(child)
+
+        # This allows restructuredtext markup to use TypeDocstring as well with a CLI option: --process-types
+        field_parsed_doc: ParsedDocstring
+        if self._processtypes and tagname in ['type', 'rtype']:
```

Comment:
We repeat two times this list: ``['type', 'rtype']``, maybe it would be good to have it stored somewhere as a constant ?